app/src/main/res/values-it/strings.xml

```diff
@@ -342,4 +342,519 @@
     <string name="successfully_invited">Invitato con successo</string>
     <string name="reply">Rispondi</string>
     <string name="failed_selecting_image">Caricamento immagine fallito.</string>
+    <string name="private_repo">Privato</string>
+
```

Comment:
Why are you adding a new line in between each string?

Reply:
@yakov116 I simply copied the strings from the English file... but in the next update I'll remove them!

_Sent from my ZenFone 3 (ZE520KL) using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_